test/service_objects/game_piece_action_resolvers/melee_attack_resolver.test.ts

```diff
@@ -130,8 +128,7 @@ describe('resolveMeleeAttackAction', () => {
           publicKey: { x: 'xValue', y: 'yValue' },
           cipherText: 'supersecret',
           signature: { r: 'rValue', s: 'sValue' }
-        },
-        resolvedAttacks: undefined
+        }
       }
```

Comment:
No need to specify undefined

Reply:
Yeah this wasn't needed. I suppose it just made it more explicit to the reader that we know `resolvedAttacks` will be a thing later but are intentionally not setting it at this time. Makes sense to just remove it though.

Reply:
Ok I put it back for now.  I'm not opposed to some explicitness!